.github/workflows/deploy.yml

```diff
@@ -0,0 +1,42 @@
+name: Trigger action for GOV.UK_co-cyber-security-external
+
+on:
+  workflow_dispatch:
+  push:
+    branches:    
+      - 'main'
+    paths:
+      - 'security.txt'
+      - 'thanks.txt'
+  schedule:
+     - cron: "0 2 * * MON-FRI" # Runs at 02:00 UTC
```

Comment:
you are missing one more star at the end of the cron - https://docs.aws.amazon.com/lambda/latest/dg/services-cloudwatchevents-expressions.html

Reply:
Many thanks and I will review this.